### ISSUE TITLE
Add scenario test coverage for WebSockets using client certificate.

### ISSUE THREAD

**Issue by StephenBonikowsky** [dotnet/wcf#506]
We have tests for both WebSockets and Client Certificates but not where both features are combined.

**Comment by StephenBonikowsky** [comment]
Currently throwing PNSE for both N and K.
Test is still needed but lower priority.

**Comment by StephenBonikowsky** [comment]
We now have 17 tests for WebSockets representing over 50% code coverage and we believe that all main and secondary scenarios are being covered, one test is still being skipped due to a bug in CoreFx.
Closing this issue.

**Comment by StephenBonikowsky** [comment]
Commented and closed the wrong issue :cry:

**Comment by zhenlan** [comment]
Move to future as this is blocked by the availability of WebSocket client on Linux (tracked by #625)

**Comment by zhenlan** [comment]
WebSocket client is supported on Linux now.

**Comment by zhenlan** [comment]
At this point, we still don't have any functional tests in WebSocket scenario using client cert.
